MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -59,7 +61,11 @@
 #include <vector>
 #include <set>
 
-#define LINE_MIN_NUMBER_POINTS 2
+// local constants
+static const int LINE_MIN_NUMBER_POINTS = 2;
+static const int MINIMUM_MARKUPS_NUMBER = 3;
```

Comment:
We usually follow VTK convention of calling "count" variables as NumberOfX. So, it should be something like NUMBER_OF_MARKUPS_MIN and NUMBER_OF_LINE_POINTS_MIN.